js/ui/widget/ui.errors.js

```diff
@@ -233,11 +233,6 @@ module.exports = errorUtils(errors.ERROR_MESSAGES, {
     */
     E1056: 'The {0} widget does not support buttons of the "{1}" type',
 
-    /**
-    * @name ErrorsUIWidgets.E1057
-    */
-    E1057: 'The {0} widget does not support asynchronous validation rules in the "{1}" edit mode',
```

Comment:
After removing the new added error will have the same error code and the customer with 19.2 version will see the wrong error.

Reply:
I'll comment it out. So, if we generate a new ID in the future, we will skip this particular index and will delete the comment.